docs/server/source/appendices/docker-on-mac.md

```diff
@@ -0,0 +1,88 @@
+# Run BigchainDB with Docker On Mac
+
+**NOT for Production Use**
+
+Those hacking on Mac can follow this document to run BigchainDB.
+Running BigchainDB on Mac(Docker or otherwise) is not officially supported
+currently.
```

Comment:
Unless Apple returns to the server market, we will not be supporting running BigchainDB Server on Macs.